### PR TITLE
Update available ElasticSearch versions

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

File: services/elasticsearch.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ To use it in your application, add it to ``.symfony/services.yaml``:
 
     # .symfony/services.yaml
     mysearch:
-        # supported versions: 6.5 and 7.2
+        # supported versions: 6.5, 7.2, 7.5, 7.6, 7.7, 7.9
```
Comment (OskarStark) [inline services/elasticsearch.rst:12]: not `7.8` ?

Comment (sgrodzicki) [reply services/elasticsearch.rst:12]: 6.8 is missing as well and it's the last minor in 6.x with EOL 2020-11-20, but maintained until 8.0.0 is available

Comment (sgrodzicki) [reply services/elasticsearch.rst:12]: 6.5 and 7.2 should be moved to the "not maintained upstream" section below per https://www.elastic.co/support/eol#elasticsearch

Comment (tgalopin) [reply services/elasticsearch.rst]: Updated

6.8 and 7.8 are indeed not supported.

Comment (sgrodzicki) [reply services/elasticsearch.rst]: 6.8 should still be listed as some users aren't ready to upgrade to 7.x

Comment (OskarStark) [reply services/elasticsearch.rst]: @sgrodzicki I think these versions are not maintained/available by/for SymfonyCloud

Comment (sgrodzicki) [reply services/elasticsearch.rst]: I was able to deploy all those listed below on SymfonyCloud:

- 0.90
- 1.4
- 1.7
- 2.4
- 5.2
- 5.4
- 6.5
- 7.2
- 6.8 (EOL 2020-11-20, but maintained until 8.0.0 is available)
- 7.5 (EOL 2021-06-02)
- 7.6 (EOL 2021-08-11)
- 7.7 (EOL 2021-11-13)
- 7.9 (EOL 2022-02-18)

**Pull request merged**